jwst/master_background/nirspec_corrections.py

```diff
@@ -21,6 +21,23 @@ def correct_nrs_ifu_bkg(input):
     """
 
     log.info('Applying point source pathloss updates to IFU background')
-    input.data *= (input.pathloss_point / input.pathloss_uniform)
+
+    # Try to load the appropriate pathloss correction arrays
+    try:
+        pl_point = input.getarray_noinit('pathloss_point')
+    except AttributeError:
+        log.warning('Pathloss_point array not found in input')
+        log.warning('Skipping pathloss background updates')
```

Comment:
It might be good to set the step status keyword to "SKIPPED" in this block and the one below.

Reply:
But the master background step itself is not being skipped in this case. Only the application of the corrections to the master bkg signal. Perhaps we should in fact skip the entire step if the corrections can't be applied.

Reply:
Ah, right.  I guess if the pathloss arrays are not there, the pathloss step was not run?  Does that mean this is just fine as is?

Reply:
That's a good point. I was thinking of the case where someone was using old input files that don't have the proper pathloss arrays attached, but this would actually properly handle the case of the pathloss step having been skipped (I think). In that case the master bkg should already match the actual bkg, so there's no need to remove the previously applied correction.

Reply:
But overall it would perhaps be best to skip the whole step, so that we don't deliver improperly calibrated results from the master bkg step. If someone has turned off pathloss upstream, they're going to get bad calibration on several fronts (i.e. flux cal won't be right, ...)

Reply:
Hmm, at that point perhaps we are getting to the point where things need to be managed by the pipeline.  If people decide to skip steps that are essential, then they do so at their own risk?  Perhaps your original approach is correct.  Just return the input, throw a big warning in the log, and be done with it.